modules/sfp_peegeepee.py

```diff
@@ -0,0 +1,205 @@
+# -*- coding: utf-8 -*-
+# -------------------------------------------------------------------------------
+# Name:        sfp_peegeepee
+# Purpose:     SpiderFoot plug-in for looking up e-mail addresses and domains
+#              on PeeGeePee.com.
+#
+# Author:      <bcoles@gmail.com>
+#
+# Created:     2019-07-07
+# Copyright:   (c) bcoles
+# Licence:     GPL
+# -------------------------------------------------------------------------------
+
+import re
+import time
+import urllib
+from sflib import SpiderFoot, SpiderFootPlugin, SpiderFootEvent
+
+
+class sfp_peegeepee(SpiderFootPlugin):
+    """PeeGeePee:Footprint,Investigate,Passive:Public Registries::Look up e-mail addresses and domains on PeeGeePee.com."""
+
+    results = dict()
+
+    # Default options
+    opts = {
+        'timeout': 15,
+        'fetch_keys': False
+    }
+
+    # Option descriptions
+    optdescs = {
+        'timeout': 'Query timeout, in seconds.',
+        'fetch_keys': 'Retrieve PGP keys for each match.'
+    }
+
+    currentEventSrc = None
```

Comment:
This is only needed when multiple parts of the module submit events and need to reference the source (`sfp_arin` is an example which does that), but this module isn't doing that so it can be removed.

Reply:
Done